modules/ignition/resources/services/kubelet.service

```diff
@@ -11,6 +11,9 @@ ExecStart=/usr/bin/hyperkube \
     --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \
     --kubeconfig=/var/lib/kubelet/kubeconfig \
     --rotate-certificates \
+    --container-runtime=remote \
+    --container-runtime-endpoint=unix:///var/run/crio/crio.sock \
+    --runtime-request-timeout=10m \
```

Comment:
Motivation for this setting is [here][1].

[1]: https://github.com/kubernetes-sigs/cri-o/blob/v1.11.3/kubernetes.md#preparing-kubelet